recipes-devtools/bats-suite/files/openxt-bats-suite.initscript

```diff
@@ -0,0 +1,22 @@
+#! /bin/sh
+set -e
+
+# Source sysvinit rcS.d defaults.
+. /etc/default/rcS
+
+# Source sysvinit functions library.
+. /etc/init.d/functions
+
+# /etc/init.d/openxt-bats-suite: run the OpenXT bats test-suite.
+
+LOG_FILE="/storage/openxt-bats-suite.log"
+
+begin "Starting OpenXT bats test-suite..."
+
+for dir in /usr/libexec/{dom0,ndvm,uivm}; do
+    if [ -d "${dir}" ]; then
+        bats "${dir}" | tee "${LOG_FILE}" | logger -t 'bats-suite'
```

Comment:
Why a separate log file if you are already sending everything to syslog? IMHO I would say either manage the logging in the script or just let syslog do it, not both

Reply:
Because I found it easier to read and, should the `syslog` fail, this still let use get the logs. I am not particularly attached to it though, I can remove the `LOG_FILE` entirely.

Reply:
This gets to a sticking point for me, I personal would like to see us split up syslog logging instead of putting everything in one file. The counter argument that is always given to me is, that is what grep is for. In that same vein, since I will likely never win the syslog split argument, IMHO I think it is probably better to just write to the file from the script and drop the logger part.

Reply:
The original idea was that by using one (logrotated) file, we know exactly how much room is available for it, and it's easy to setup things like shutting down the platform when /var/log is about to be full.
If more files are sharing the space, there might be some concurrent access issues to the available disk space...

Reply:
There is no reason doing multiple files would not work with duplicating the current logrotate wrapper pipe that is in use today. There is nothing in place today that I am aware of that shuts down the platform when `/var/log` is full. If that is desired, an easy approach is to use `auditd` with shutdown policy.

Reply:
Using the `syslog` for now is trivial. Since this change-set is about bringing an automated test suite capability, I would prefer tackling the `syslog` in a separate PR if that's ok.

Reply:
Ack